### DISCUSSION TITLE
iOS: When exiting the app, the privacy screen does not come up.

### DISCUSSION THREAD

**Discussion by RayRayDevDev** [ente-io/ente#961] [Enhancements]
When exiting the iOS app with Face ID enabled, the expected behaviour is that the app should have a privacy screen enabled such that, upon reentry into the app (without completely closing it), the photo gallery is blocked by the privacy screen until Face ID authentication is completed. 

What I've observed is that while Face ID does indeed activate upon switching apps, there's no privacy screen activated at the same time, such as with Standard Notes, thereby showing any person who happens to have my phone, unauthenticated, whatever I was last viewing in my gallery, which therefore mitigates the purpose of the Face ID authentication in the first place. 

I have the latest App Store release installed on an iPhone 13 Pro. 

Thanks!

**Comment by vishnukvmd** [comment]
Hey @RayRayDevDev, thanks for creating the issue, this is a known limitation with the plug-in we're currently using for the lockscreen. We'll figure out a workaround.

**Reply by mrmurphy** [reply]
Throwing my vote in here also. This privacy is key to me. It’s also an issue that there’s no privacy cover in the app switcher on iPhone or iPad. If I’m viewing private photos, then leave the app, then open the app switcher, those private photos will show in the app screenshot.

I’d love to see this one addressed ASAP. Thanks!
